### DISCUSSION TITLE
Ability To Flag Tracks As Forced ?

### DISCUSSION THREAD

**Discussion by Jamiex304** [MikeMoolenaar/MkvDefaultTrackChanger#8] [Ideas]
Found this tool and its been super useful to helping sort my media library and set tracks up correctly.

Wondering though if it would be possible to a flag or and second option to for example flag tracks are forced, instead of straight up default?

This would be useful for anime content, where a lot of the time there exists a signs & songs track and a subtitles English track

**Comment by MikeMoolenaar** [comment]
Glad you found it useful so far! I'm not sure I get what you mean, do you mean setting the track to forced instead of default? What is the advantage of doing that?
